apps/blackjack-client/src/main/java/GameContext/TrumpCard/TrumpCardController.java

```diff
@@ -16,28 +16,28 @@ public TrumpCardController() {
-        CARDS.put("undoHit", new TrumpCardPOJO("Remove my last card", "Remove your latest card drawn.", "undoHit", "UTILITY", new ImageIcon("resources/UndoHit.PNG")));
-        CARDS.put("invincibility", new TrumpCardPOJO("Protect from opponent's ATTACK trump cards", "Protect you from being attack by opponent's ATTACK trump cards in this round.", "invincibility", "UTILITY", new ImageIcon("resources/Invincibility.PNG")));
+        CARDS.put("seeNextHit", new TrumpCardPOJO("saw next 2 cards from the deck.", "Player will be notified what the next 2 cards in being draw next.", "seeNextHit", "UTILITY", new ImageIcon("resources/SeeNextHit.PNG")));
+        CARDS.put("changePointsLimit25", new TrumpCardPOJO("change point target to 25.", "Change game win condition to 25 instead of 21.", "changePointsLimit25", "UTILITY", new ImageIcon("resources/ChangePointLimit25.PNG")));
+        CARDS.put("undoHit", new TrumpCardPOJO("remove their last card.", "Remove your latest card drawn. *Does not effect opponent first 2 cards*", "undoHit", "UTILITY", new ImageIcon("resources/UndoHit.PNG")));
+        CARDS.put("invincibility", new TrumpCardPOJO("cleanse and protect themself from status and effect.", "Protect you from being attack by opponent's ATTACK trump cards / cleanse all your negative status in this round.", "invincibility", "UTILITY", new ImageIcon("resources/Invincibility.PNG")));
 
 //        TrumpCard's status
-        STATUSES.put("INVINCIBILITY", new TrumpStatusPOJO("Protect from opponent's ATTACK trump cards", "Protect you from being attack by opponent's ATTACK trump cards in this round.", "INVINCIBILITY",  new ImageIcon("resources/Invincible_Status.PNG")));
-        STATUSES.put("BLIND", new TrumpStatusPOJO("Enemy can't see their next hit card", "Enemy can't see their next hit card.", "BLIND",  new ImageIcon("resources/Blind_Status.PNG")));
-        STATUSES.put("DENY_HIT", new TrumpStatusPOJO("Opponent cannot hit in this round", "Opponent cannot hit in this round.", "DENY_HIT",  new ImageIcon("resources/Deny_hit_Status.PNG")));
-        STATUSES.put("DENY_TRUMP_USE", new TrumpStatusPOJO("Opponent cannot use their trump cards", "Opponent cannot use their trump cards.", "DENY_TRUMP_USE",  new ImageIcon("resources/Deny_Trump_Use_Status.PNG")));
+        STATUSES.put("INVINCIBLE", new TrumpStatusPOJO("INVINCIBLE", "Protect you from being attack by opponent's ATTACK trump cards in this round.", "INVINCIBILITY",  new ImageIcon("resources/Invincible_Status.PNG")));
+        STATUSES.put("BLIND", new TrumpStatusPOJO("BLIND", "Opponent can't see any cards for one round.", "BLIND",  new ImageIcon("resources/Blind_Status.PNG")));
+        STATUSES.put("DENY_HIT", new TrumpStatusPOJO("DENY HIT", "Opponent cannot hit in this round.", "DENY_HIT",  new ImageIcon("resources/Deny_hit_Status.PNG")));
```

Comment:
ไม่แน่ใจว่าเวลาเอา mouse hover status แล้วจะแสดงคำพูดเหมาะสมไหมเช่น

หาก hover ตัวเองจะได้ว่า
Opponent cannot hit in this round.

อาจจะเปลี่ยนเป็นอะไรที่แสดงได้ทั้งสองฝ่ายเช่น
The player cannot perform hit this round.